apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts

```diff
@@ -42,17 +43,18 @@ export class NotificationsController {
         locale: {
           required: false,
           type: 'string',
-          example: 'is-IS',
+          description: 'locale',
+          example: 'en',
```

Comment:
This isn't that important but isn't the type an enum? Can you just pick an enum value then?
Not a stopper, just a thought :)